contracts/modules/common/BaseModule.sol

```diff
@@ -80,6 +80,17 @@ contract BaseModule is IModule {
         _;
     }
 
+    /**
+    * @dev Utility method enbaling anyone to recover ERC20 token sent to the
+    * module by mistake and transfer them to the Module Registry.
+    * @param _token The token to recover.
+    */
+    function recoverToken(address _token) external override {
+        uint total = ERC20(_token).balanceOf(address(this));
+        bool success = ERC20(_token).transfer(address(registry), total);
```

Comment:
Maybe our 2.0 upgrade is a good opportunity to also enable recovery of non-compliant ERC20 tokens. See https://medium.com/coinmonks/missing-return-value-bug-at-least-130-tokens-affected-d67bf08521ca 
We could for example use OpenZeppelin's `SafeERC20.safeTransfer` or something simpler.

Reply:
Valid point. Can you add it to trello? Trying to keep the upgrade PR focussed.

Reply:
Done